content/event_info.md

```diff
@@ -1,9 +1,9 @@
 <div markdown="1" class="column is-6">
 <div markdown="1" class="content">
 
-Started in 2016, WASD (Warwick's Awesome Speedruns and Demos) is a 2-day speedrunning event run annually at the University of Warwick. It is the largest student-run speedrunning event in the UK. Past events have featured a diverse range of games, from Final Fantasy IX to Rise of the Tomb Raider, from Super Monkey Ball to Parasite Eve II. WASD 2021 will run on the 13th and 14th February 2021. The event schedule has been finalised and can be found [here](https://oengus.io/marathon/wasd-2021/schedule).
+Started in 2016, Warwick's Awesome Speedruns and Demos (WASD) is a 2-day speedrunning event held annually at the University of Warwick. It is the largest student-run speedrunning event in the UK. Past events have featured a diverse range of games, from Final Fantasy IX to Rise of the Tomb Raider, from Super Monkey Ball to Parasite Eve II. WASD runs over the course of a weekend each February, and WASD Summer, a new event, will run between each main WASD event during the summer. The first WASD Summer will be held on 14-15th August 2021.
 
-Donations to the event are in support of [SpecialEffect](https://specialeffect.org.uk/), a wonderful UK-based charity whose mission is to put fun and inclusion back into the lives of people with physical disabilities, by helping them to play video games. They do this by using and developing a range of accessibility hardware and software, from modified joypads to eye-gaze technology. The WASD events to date have raised over £5,000 for SpecialEffect!
+Donations to the event are in support of [SpecialEffect](https://specialeffect.org.uk/), a wonderful UK-based charity whose mission is to put fun and inclusion back into the lives of people with physical disabilities, by helping them to play video games. They achieve this by using and developing a range of accessibility hardware and software, from modified joypads to eye-gaze technology. To date, the WASD events have raised over £5,000 for SpecialEffect!
```

Comment:
SD: I am not sure about the amount now and it's still _technically_ £5k+ but I think we're above £6k raised now. It's not an issue though.